assets/javascripts/discourse/initializers/checklist.js

```diff
@@ -100,13 +103,17 @@ export function checklistSyntax($elem, postDecorator) {
           });
 
           if (save && save.then) {
-            save.then(() => {
-              postWidget.attrs.isSaving = false;
-              postWidget.scheduleRerender();
-            });
+            save
+              .then(() => {
+                postWidget.attrs.isSaving = false;
```

Comment:
I no you didn't introduce this, but not a fan of this pattern where we mutate input data